### DISCUSSION TITLE
Persist form data when changing page

### DISCUSSION THREAD

**Discussion by YuriPadilla** [neuefische/questions#502] [Web] [answered]
Hello everyone!

I want the form data for my WebApp to persist when changing the page. The question is: should I use local storage for it? ...or global state with zustand??? ...or jotai???

Thanks a lot!!! 👍

**Comment by ThomasF85** [comment, marked as answer]
Hi Yuri,

it depends on what behavior you are after.
If you want the user input to "persist" in the form even after closing the browser and opening it again at a later time, then you should go with localStorage.
If the user input should only persist as long as the user does not reload the application manually or close the browser window, then global state is sufficient and zustand and jotai will both work.

**Reply by YuriPadilla** [reply]
Thanks Thomas!
I decided to use jotai.
👍
